docs/FAQ.md

```diff
@@ -0,0 +1,52 @@
+### Frequently Asked Questions
+
+#### Table of Contents
+* [What is KUDO?](#what-is-kudo)
+* [When would you use KUDO?](#when-would-you-use-kudo)
+* [What is a Framework](#what-is-a-framework)
+* [What is a Frameworkversion?](#what-is-a-frameworkversion)
+* [What is an Instance?](#what-is-an-instance)
+* [What is a Planexecution?](#what-is-a-planexecution)
+* [What is a Parameter?](#what-is-a-parameter)
+* [What is a Trigger?](#what-is-a-trigger)
+* [When I create a Framework, it is automatically creating new CRDs?](#when-i-create-a-framework-it-is-automatically-creating-new-crds)
+* [How does it work from a RBAC perspective?](#how-does-it-work-from-a-rbac-perspective)
+* [Is the dependency model an individual controller per workload?](#is-the-dependency-model-an-individual-controller-per-workload)
+
+#### What is KUDO?
+
+KUDO is an operator we are building that is specifically designed to help provide operations to operators. We want to capture the actions that are required for managing applications in production as part of the deployment for the applications themself. To embed those best practices in the operator CRD. So KUDO is the operator that handles those specs that are provided as declarative CRDs.
```

Comment:
~deployment~ definition
I want to focus on the "binary" containing these operations, not focused on an application operator defining them in the deployment.  The AO will have them provided to them in the CRDs